content/components/action-menu.mdx

```diff
@@ -8,40 +8,65 @@ import {Box, Button, Heading, Link} from '@primer/react'
 export default ComponentLayout
 
-## Usage
+<Note>Multi level is not available yet in the React implementation.</Note>
 
-Action menu is used for actions, navigation, to display secondary options, or single/multi select lists. They appear when users interact with buttons, actions, or other controls.
+## Usage
 
-## Anatomy
+An action menu is a list of items, with each item representing an action, command, or current selection, which can be a single or multi-select.
 
 <img
   width="960"
-  alt="Action menu opened with a label for the trigger (button), overlay (menu), and the internal action list."
-  src="https://user-images.githubusercontent.com/18661030/226487158-06c6cc38-65de-438c-8028-373862fa0dfe.png"
+  alt=""
+  src="https://user-images.githubusercontent.com/980622/233374605-cee72894-fc5e-46f4-a1ea-e82aa26084ef.png"
 />
 
-## Options
+There are different ways to use menus, such as:
 
-<img
-  src="https://user-images.githubusercontent.com/18661030/226486461-c43f75c5-d9ed-415f-907c-9d2dbf71e3a2.png"
-  width="960"
-  alt=""
-/>
-<Caption>
-  The trigger button can be a standard or icon button. For single select menus that display the selection in the
-  trigger, a label must be persist either internally or externally.
-</Caption>
+- **Button**: Shows items directly related to the current view.
+- **Button (Selection)**: Shows items directly related to the button, with a title prefix, like "Sort by: Title."
+- **Context menu**: Often contains multiple levels and tends to be only available through right clicking items or through [icon buttons](/components/icon-button) that appear on hover.
+
+It's important to note that each menu item should have consistent and limited interactions:
+
+- `enter`, `click` or `tap`: Triggers an action or navigates into a submenu
+- `arrow left`, `arrow up`, `arrow down`, `arrow right`: Navigates the menu
+
+Menus are a commonly used feature across multiple platforms, and both [Apple](https://developer.apple.com/design/human-interface-guidelines/components/menus-and-actions/menus) and [Microsoft](<https://learn.microsoft.com/en-us/previous-versions/windows/desktop/bb226797(v=vs.85)>) have detailed documentation on their menus. To create a pleasant user experience, it's important to follow their expected behavior closely.
+
+## Anatomy
 
 <img
-  src="https://user-images.githubusercontent.com/18661030/226486463-893231bb-8f57-4255-83a4-7d468b2be5ed.png"
   width="960"
-  alt=""
+  alt="Action menu opened with a label for the trigger (button), overlay (menu), and the internal action list."
+  src="https://user-images.githubusercontent.com/980622/234074196-f67bf01e-29b6-4be1-a6fc-47fc532f536b.png"
 />
-<Caption>Menus can contain simple actions and links, single select items, or multi select items.</Caption>
 
-## Behavior
+## Content
+
+### Trigger
 
-### Opening
+Action menus can be opened through a [button](/components/button), [icon button](/components/icon-button) or by right clicking items.
```

Comment:
> by right clicking items

I know this is a new pattern we are working on introducing, but what is an "item"? Could we be more specific here? Or is this not the right place to be specific 🤔 do you mean action list item?

Reply:
Updated it to `right clicking list items`. I reckon it's less likely to be used on action list items than generic list items or data table items.